go.mod

```diff
@@ -4,86 +4,84 @@ go 1.15
 
 require (
 	github.com/akutz/gofsutil v0.1.2
-	github.com/container-storage-interface/spec v1.2.0
+	github.com/container-storage-interface/spec v1.3.0
```

Comment:
any particular reason we are picking `v1.3.0`?
v1.3.0 is released a year before (Jun 3, 2020).

we have v1.4.0 released in Mach 2021 - https://github.com/container-storage-interface/spec/releases/tag/v1.4.0

Reply:
+1

Reply:
Updated to v1.4.0.
Initially it was automatically updated to v1.3.0 during `make check` probably because of some 1.21 dependency.